modules/core/src/core/animation-loop-proxy.js

```diff
@@ -121,23 +121,21 @@ export default class AnimationLoopProxy {
   /* Public methods */
 
   // Starts a render loop if not already running
-  start(opts = {}) {
+  async start(opts = {}) {
```

Comment:
This is technically a breaking change (returning self vs. returning promise). Add to doc?